src/main/java/utils/BrowserManagement.java

```diff
@@ -52,7 +52,7 @@ public static WebDriver initializeDriver() {
                 ChromeOptions chromeOptions = new ChromeOptions();
                 chromeOptions.addArguments("--start-maximized");
                 chromeOptions.addArguments("--incognito");
-                chromeOptions.setHeadless(true);
+                chromeOptions.setHeadless(false);
```

Comment:
Headless mode is the default. You can remove this line.

Reply:
ok